lib/api/model/initial_snapshot.dart

```diff
@@ -84,6 +84,7 @@ class InitialSnapshot {
     required this.unreadMsgs,
     required this.streams,
     required this.userSettings,
+    required this.realmDefaultExternalAccounts,
```

Comment:
The tree isn't clean on rerunning `dart run build_runner build`, because the generated file has these in a different order.